<launch>

  <node pkg="learned_map" type="sessionFilter" name="sessionFilter" output="screen">

  <param name="loadMapName" value="/home/yh/mapModel/12.25/wholeMap_s10.vtk"/>
  <param name="saveMapName" value="/home/yh/mapModel/05.09/salient.vtk"/>
  <param name="staticInt" value="280"/> 

  <param name="ifCluster" value="true"/>
  <param name="rangeSearch" value="3"/>	
  <param name="minStaticPoints" value="5"/>
  <param name="kSearch" value="20"/>


  </node>
<node pkg="rviz" type="rviz" name="rviz" args="-d $(find learned_map)/cfg/sessionFilter.rviz" output="screen"/>
</launch>
